### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by Aya92syr**

### CODE REVIEW

File: Week1/exercise/app.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+'use strict';
```
Comment (remarcmij) [inline Week1/exercise/app.js]: This exercise file is not part of the homework and should not be in your pull request.

File: Week1/exercise/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (remarcmij) [inline Week1/exercise/index.html]: Ditto here.

File: Week1/homework/app.js [hunk 3]
```diff
@@ -8,4 +8,5 @@
```
Comment (remarcmij) [inline Week1/homework/app.js]: This (unfinished) file from week1 should not be in your pull request for week2.

File: Week2/homework/live/exercise.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
+'use strict';
```
Comment (remarcmij) [inline Week2/homework/live/exercise.js]: This file is not part of the week2 homework and should not be in your pull request.

File: Week2/homework/maartjes-work.js [hunk 5]
```diff
@@ -46,16 +46,22 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
```
Comment (remarcmij) [inline Week2/homework/maartjes-work.js]: The answer produced by your code is incorrect.

**Comment by Aya92syr** [comment]
Hello Jim,

Thank you for your feedback
As for what you asked us, I was not in the group yet, I will correct all
the mistakes

Regards,
Aya

‫في الأحد، 20 يناير 2019 في 10:42 م تمت كتابة ما يلي بواسطة ‪Jim Cramer‬‏ <‪
notifications@github.com‬‏>:‬

**Comment by remarcmij** [comment]
Oh, I see. If I remember correctly you were present during the first lecture, but you were not (yet) part of the @studentsclass19 slack group I think, so that is probably why you missed the message. I'm looking forward to your corrections (and your week1 and later week3 homework).

**Comment by remarcmij** [comment]
Hi @Aya92syr, any progress on this?

**Pull request closed without merging**